proto/autokitteh/connections/v1/svc.proto

```diff
@@ -64,4 +82,6 @@ service ConnectionsService {
   rpc Update(UpdateRequest) returns (UpdateResponse);
   rpc Get(GetRequest) returns (GetResponse);
   rpc List(ListRequest) returns (ListResponse);
+  rpc Test(TestRequest) returns (TestResponse);
+  rpc RefreshStatus(RefreshStatusRequest) returns (RefreshStatusResponse);
```

Comment:
Please document what's the difference between these 2 functions that have the same input and the same output.

Reply:
done